types/ProductionAccess.ts

```diff
@@ -19,8 +19,8 @@ export interface ProductionAccessSupportEmail {
   monitizationExplanation?: string;
   veteranFacing?: boolean;
   website?: string;
-  signUpLink?: string[];
-  supportLink?: string[];
+  signUpLink?: string;
```

Comment:
impetus?

Reply:
Looks like it got changed to string and changed back to array and I think those changes got lost in the merge shuffle. Good catch. Fixed!